CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Getting started
+
+The OST repository has two purposes:
+- Maintaining the OST website
+- Storing documentation related to the team goals and strategy
+
+# Process
+
+## Create issue and PR
+
+- Fork the repository
+- Create an issue
+- Open a PR and use the label - `pr: Work in Progress` (See below [template](#Example template for PR))
```

Comment:
Link syntax is incorrect so it does not correctly link to the header further down